docs/pages/develop/user-interface/splash-screen-and-app-icon.mdx

```diff
@@ -4,153 +4,123 @@ description: Learn how to add a splash screen and app icon to your Expo project.
+Open the app config file, under plugins set the corresponding properties as follows:
 
 ```json app.json
 {
-  "splash": {
-    /* @info */
-    "image": "./assets/images/splash.png"
-    /* @end */
+  "expo": {
+    "plugins": [
+      [
+        "expo-splash-screen",
+        {
+          /* @info The background color of the splash screen */
+          "backgroundColor": "#232323",
+          /* @end */
+          /* @info The path to the splash screen icon */
+          "image": "./assets/splash-icon.png",
+          /* @end */
+          /* @info The path to the splash screen icon for dark mode (optional) */
+          "dark": {
+            "image": "./assets/splash-icon-dark.png",
+            /* @end */
+            /* @info The background color of the splash screen for dark mode */
+            "backgroundColor": "#000000"
+            /* @end */
+          },
+          /* @info The width of the splash screen icon */
+          "imageWidth": 200
+          /* @end */
+        }
+      ]
+    ]
   }
 }
 ```
 
-You can test your new splash image by starting the development server with `npx expo start` and using a development build or Expo Go to preview it.
+To test your new splash screen, create a [preview](/build/eas-json/#preview-builds) or [production](/build/eas-json/#production-builds) build of your app.
 
-> **Note:** In development mode, when you test your new splash screen, you will notice an information bar at the bottom of the screen. It displays the information about how much of the JavaScript code is downloaded on the device when preparing your app. It doesn't appear in production apps.
+> **warning** Do not use Expo Go to test your splash screen as Expo Go renders your app icon while the splash screen is visible. Instead, use a [preview build](/build/eas-json/#preview-builds) or a [production build](/build/eas-json/#production-builds).
 
 </Step>
 
-<Step label="4">
+<Collapsible summary={<>Configuring <CODE>expo-splash-screen</CODE> properties separately for Android and iOS</>}>
 
-### Change the background color
-
-If you set a background color other than white for your splash image, you may see a white border around it. This is due to the [`splash.backgroundColor`](/versions/latest/config/app/#backgroundcolor-3) property that has a default value of `#ffffff`.
-
-You can update this property's value to match your splash image's background color. For example:
+[expo-splash-screen](/versions/latest/sdk/splash-screen) also supports `ios` and `android` properties for configuring the splash screen for a specific platform. See the following example:
 
 ```json app.json
 {
-  "splash": {
-    "image": "./assets/images/splash.png",
-    /* @info */
-    "backgroundColor": "#FEF9B0"
-    /* @end */
+  "expo": {
+    "plugins": [
+      [
+        "expo-splash-screen",
+        {
+          "ios": {
+            "backgroundColor": "#ffffff",
+            "image": "./assets/splash.png",
```

Comment:
```suggestion
            "image": "./assets/images/splash-icon.png",
```

Let's keep the path of these icons as per default template directory structure. For new users it might lead to confusion.